### ISSUE TITLE
Support for Hotjar Recording Tags

### ISSUE THREAD

**Issue by AhmedHS** [maxandriani/ngx-hotjar#2]
First of all, thanks for writing this! It's been super useful so far!

I was curious if we could add support for tagging recordings in Hotjar.

The documentation for this is available at: https://help.hotjar.com/hc/en-us/articles/115011819488-How-to-Tag-your-Hotjar-Recordings

The code would look something like so:
```
/**
   * Allows the user to add custom tags to an an ongoing recording.
   *
   * @param path Array of tags
   */
  tagRecording(path: string[]): void {
    try {
      hj('tagRecording', path);
    } catch (err) {
      this.error(err);
    }
  }
```

Thanks again!

**Comment by maxandriani** [comment]
@AhmedHS Thanks for bring me this! I'll add this feature as soon as possible. But feel free to open a PR if you need urgency.

**Comment by maxandriani** [comment]
Published on release 0.0.5

**Comment by AhmedHS** [comment]
Thanks @maxandriani!
